step-01/js/main.js

```diff
@@ -1,26 +1,23 @@
 'use strict';
 
-navigator.getUserMedia = navigator.getUserMedia ||
-    navigator.webkitGetUserMedia || navigator.mozGetUserMedia;
-
-var constraints = {
-  audio: false,
-  video: true
+// On this codelab, you will be streaming only video (video: true).
+const mediaStreamConstraints = {
+  video: true,
 };
 
-var video = document.querySelector('video');
+// Video element where stream will be placed.
+const localVideo = document.querySelector('video');
 
-function successCallback(stream) {
-  window.stream = stream; // stream available to console
-  if (window.URL) {
-    video.src = window.URL.createObjectURL(stream);
-  } else {
-    video.src = stream;
-  }
-}
+// Handles success by adding the MediaStream to the video element.
+const gotLocalMediaStream = function(mediaStream) {
```

Comment:
I would stick with the syntax `function gotLocalMediaStream(mediaStream)`.

I don't think there's any benefit here of using an anonymous function expression (is there?) It just makes the code more verbose.